src/main/java/io/github/terra121/util/CardinalDirection.java

```diff
@@ -1,21 +1,23 @@
 package io.github.terra121.util;
 public enum CardinalDirection {
-	N (337.5, 22.5, "North"),
-	S (157.5, 202.5, "South"),
-	E (67.5, 112.5, "East"),
-	W (247.5, 292.5, "West"),
-	NW (292.5, 337.5, "Northwest"),
-	SW (202.5, 247.5, "Southwest"),
-	SE (112.5, 157.5, "Southeast"),
-	NE (22.5, 67.5, "Northeast"),
-	UNKNOWN (360.5, 10000000.0, "Unknown");
+	N (337.5, 22.5, "North", "terra121.cardinal_directions.north"),
+	S (157.5, 202.5, "South", "terra121.cardinal_directions.south"),
+	E (67.5, 112.5, "East", "terra121.cardinal_directions.east"),
+	W (247.5, 292.5, "West", "terra121.cardinal_directions.west"),
+	NW (292.5, 337.5, "Northwest", "terra121.cardinal_directions.northwest"),
+	SW (202.5, 247.5, "Southwest", "terra121.cardinal_directions.southwest"),
+	SE (112.5, 157.5, "Southeast", "terra121.cardinal_directions.southeast"),
+	NE (22.5, 67.5, "Northeast", "terra121.cardinal_directions.northeast"),
+	UNKNOWN (360.5, 10000000.0, "Unknown", "terra121.cardinal_directions.north");
     private final double min;
     private final double max;
     private final String realName;
-    private CardinalDirection(double min, double max, String realName) {
+	private final String translationKey;
+    private CardinalDirection(double min, double max, String realName, String translationKey) {
         this.min = min;
         this.max = max;
         this.realName = realName;
+        this.translationKey = translationKey;
     }
     public double getMin() {
```

Comment:
these simple getters can be eliminated by putting a lombok `@Getter` annotation at the head of the class